jni/pom.xml

```diff
@@ -1,57 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.horizen</groupId>
-	<artifactId>zendoo-sc-cryptolib</artifactId>
-	<version>0.2-SNAPSHOT</version>
-	<inceptionYear>2020</inceptionYear>
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
-		<scala.version>2.12.8</scala.version>
-	</properties>
-
-	<dependencies>
-
-		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.12</version>
-			<scope>test</scope>
-		</dependency>
-
-	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.8.0</version>
-				<configuration>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-dependency-plugin</artifactId>
-				<version>3.1.1</version>
-				<executions>
-					<execution>
-						<id>copy-dependencies</id>
-						<phase>package</phase>
-						<goals>
-							<goal>copy-dependencies</goal>
-						</goals>
-						<configuration>
-							<outputDirectory>${project.build.directory}/lib</outputDirectory>
-							<overWriteReleases>false</overWriteReleases>
-							<overWriteSnapshots>false</overWriteSnapshots>
-							<overWriteIfNewer>true</overWriteIfNewer>
-							<includeScope>compile</includeScope>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-	</build>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.horizen</groupId>
+  <artifactId>zendoo-sc-cryptolib</artifactId>
+  <version>0.2-SNAPSHOT</version>
```

Comment:
We should avoid using `-SNAPSHOT` in the versions we want to publish. Let's name it `0.3.0` or even `1.0.0`. @albertog78  what do you think about this?

Reply:
The release procedure should automatically release it as 0.2 and the commit the new pom with version 0.3-SNAPSHOT. @cronicc @lander86

Reply:
@i-Alex @albertog78 

In the CI pipeline we check that git tag == pom.project.version. pom.project.version has to be a non SNAPSHOT version in order for the bundle to be published to maven central, any SNAPSHOT versions will be published to the sonatype snapshot repository at https://oss.sonatype.org/content/repositories/snapshots.

In any case publishing to maven central or the SNAPSHOT repo is only triggered if a git tag exists.

As a general rule doing a release should be it's own pull request, where the version number is bumped and separate review is being done, but lets skip this this time around. I'll set version to 0.3.0.